backend/Tim.Backend/Models/KustoQuery/KustoQueryRun.cs

```diff
@@ -40,38 +57,33 @@ public KustoQueryRun()
         {
         }
 
-        /// <summary>
-        /// Query Run states.
-        /// </summary>
-        public enum QueryRunStates
-        {
-#pragma warning disable SA1602 // Enumeration items should be documented
-            Created,
-            Completed,
-            Error,
-            TimedOut,
-#pragma warning restore SA1602 // Enumeration items should be documented
-        }
+        /// <inheritdoc/>
+        public string Id => QueryRunId.ToString();
```

Comment:
I think it would be better to either have all the guids be guids or all of them be strings vs this back and forth of having to adjust them.

Reply:
Hrmmm, yeah it's a good point.
The advantage of guids is the endpoint looks nice in swagger, with auto generated guids.
But not every database supports guids (including Kusto), which is why we need to convert to string. I really should convert just before entering it in the database I feel.